### PR TITLE
Convert the last uses of CLITestCase

### PR DESCRIPTION
**Pull request by mshriver**
@lpramuk , @ColeHiggins2  @devendra104 @Gauravtalreja1 @omkarkhatavkar I've requested your reviews because you own these test modules.

I've converted some test modules that were still using CLITestCase, and removed the no longer used test case.

Some of these tests were already not passing in master.  I tried to keep test functionality and coverage the same.

# Test Results

## test_abrt.py

All stubbed, module collects/runs without issue.

## test_hammer.py

`tests/foreman/cli/test_hammer.py::HammerCommandsTestCase::test_positive_all_options` is failing in master branch.

```
tests/foreman/cli/test_hammer.py::test_positive_all_options FAILED                                                                                                                                                [ 33%]
tests/foreman/cli/test_hammer.py::test_positive_disable_hammer_defaults PASSED                                                                                                                                    [ 66%]
tests/foreman/cli/test_hammer.py::test_positive_check_debug_log_levels PASSED                                                                                                                                     [100%]
```

## test_installer.py

All stubbed, module collects/runs without issue.

## test_sso.py

All stubbed, module collects/runs without issue.

## test_subscription.py

`tests/foreman/cli/test_subscription.py::SubscriptionTestCase::test_positive_manifest_refresh` is failing in master branch.

```
tests/foreman/cli/test_subscription.py::test_positive_manifest_upload PASSED                                                                                                                                      [ 14%]
tests/foreman/cli/test_subscription.py::test_positive_manifest_delete PASSED                                                                                                                                      [ 28%]
tests/foreman/cli/test_subscription.py::test_positive_enable_manifest_reposet PASSED                                                                                                                              [ 42%]
tests/foreman/cli/test_subscription.py::test_positive_manifest_history PASSED                                                                                                                                     [ 57%]
tests/foreman/cli/test_subscription.py::test_positive_manifest_refresh FAILED                                                                                                                                     [ 71%]
tests/foreman/cli/test_subscription.py::test_positive_subscription_list PASSED                                                                                                                                    [ 85%]
tests/foreman/cli/test_subscription.py::test_positive_delete_manifest_as_another_user PASSED                                                                                                                      [100%]
```

### CODE REVIEW

File: pytest_fixtures/api_fixtures.py [hunk 1]
```diff
@@ -56,6 +56,11 @@ def default_location():
     return entities.Location().search(query={'search': f'name="{DEFAULT_LOC}"'})[0]
 
 
+@pytest.fixture(scope='function')
```
Comment (JacobCallahan) [inline pytest_fixtures/api_fixtures.py:59]: explicit better than implicit?

Comment (mshriver) [reply pytest_fixtures/api_fixtures.py:59]: maybe I should add a comment `# ITS FUNCTION SCOPED BRO`

File: tests/foreman/cli/test_hammer.py [hunk 2]
```diff
@@ -83,145 +84,131 @@ def _format_commands_diff(commands_diff):
...
+
+    :id: 029c80f1-2bc5-494e-a04a-7d6beb0f769a
+
+    :expectedresults: Verify enabled debug log level
+
+    :CaseImportance: Medium
+
+    :BZ: 1760773
+    """
+    Admin.logging({"all": True, "level-debug": True})
```
Comment (JacobCallahan) [inline tests/foreman/cli/test_hammer.py:204]: quite a few double-quote string in this file

File: tests/foreman/cli/test_hammer.py [hunk 3]
```diff
@@ -83,145 +84,131 @@ def _format_commands_diff(commands_diff):
...
+        command_options = {option['name'] for option in output['options']}
+        command_subcommands = {subcommand['name'] for subcommand in output['subcommands']}
+        expected = fetch_command_info(command)
+        expected_options = set()
+        expected_subcommands = set()
+
+        if expected is not None:
+            expected_options = {option['name'] for option in expected['options']}
+            expected_subcommands = {subcommand['name'] for subcommand in expected['subcommands']}
+        if is_open('BZ:1666687'):
```
Comment (JacobCallahan) [inline tests/foreman/cli/test_hammer.py:109]: it looks like this was fixed back in 6.7, would you remove it?

File: tests/foreman/cli/test_subscription.py [hunk 4]
```diff
@@ -37,285 +34,247 @@
...
+
+    :CaseImportance: Critical
+    """
+    upload_manifest(function_org.id, manifests.original_manifest())
+    Subscription.list({'organization-id': function_org.id}, per_page=False)
+    Subscription.refresh_manifest({'organization-id': function_org.id})
+    Subscription.delete_manifest({'organization-id': function_org.id})
+
+
+@pytest.mark.skip_if_open("BZ:1686916")
```
Comment (JacobCallahan) [inline tests/foreman/cli/test_subscription.py:158]: this was also closed in 6.7, so should be safe to remove this check

**Review by JacobCallahan — CHANGES_REQUESTED** [review]
small things, nice!

**Review by JacobCallahan — APPROVED** [review]

**Review by omkarkhatavkar — APPROVED** [review]
ACK

**Pull request merged**